website/docs/r/independent_disk.html.markdown

```diff
@@ -45,8 +45,8 @@ resource "vcd_vapp_vm" "web2" {
 
 The following arguments are supported:
 
-* `org` - (Optional; *v2.0+*) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
-* `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
```

Comment:
Why have you removed the `v2.0+` tag? Given the amount of changes happening in the 2.0, 2.1 and next release, I'd like to keep using these tags in documentation, so for the users, who have not yet upgraded, it's easy to spot the new features.

Reply:
But independent disk resource wasn't there before 2.1 at all.